PVList.md

```diff
@@ -0,0 +1,303 @@
+
+## `$(AREA):$(DIS)-PciHub:`
+<details>
+<summary>sensors</summary>
+    Temp_DCDC{-Mon|Prs-Cte} <br>
+    Temp_DCDC_Max{-Mon|Prs-Cte} <br>
+    Temp_Switch{-Mon|Prs-Cte} <br>
+    Temp_Switch_Max{-Mon|Prs-Cte} <br>
+    Temp_PCB{-Mon|Prs-Cte} <br>
+    Temp_PCB_Max{-Mon|Prs-Cte} <br>
+    Volt_0V9{-Mon|Prs-Cte} <br>
+    Volt_0V9_Analog{-Mon|Prs-Cte} <br>
+    Volt_1V8{-Mon|Prs-Cte} <br>
+    Volt_3V3{-Mon|Prs-Cte} <br>
+    Volt_12V{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    MState-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+## `$(AREA):$(DIS)-RtmCpu:`
+<details>
+<summary>sensors</summary>
+    Temp_PCB{-Mon|Prs-Cte} <br>
+    Temp_PCB_Max{-Mon|Prs-Cte} <br>
+    Temp_DCDC{-Mon|Prs-Cte} <br>
+    Temp_DCDC_Max{-Mon|Prs-Cte} <br>
+    Temp_ComEX_CPU{-Mon|Prs-Cte} <br>
+    Temp_ComEX_PCH{-Mon|Prs-Cte} <br>
+    Temp_ComEX_DDR{-Mon|Prs-Cte} <br>
+    Volt_1V0{-Mon|Prs-Cte} <br>
+    Volt_1V8{-Mon|Prs-Cte} <br>
+    Volt_3V3{-Mon|Prs-Cte} <br>
+    Volt_5V{-Mon|Prs-Cte} <br>
+    Volt_VBAT{-Mon|Prs-Cte} <br>
+    Volt_12V_Int{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    MState-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+# Cooling Unit
+
+## `$(AREA):$(DIS)-CU_{Top|Bot}:`
+<details>
+<summary>sensors</summary>
+    Temp_LM75_1{-Mon|Prs-Cte} <br>
+    Temp_LM75_2{-Mon|Prs-Cte} <br>
+    Volt_3V3{-Mon|Prs-Cte} <br>
+    Volt_12V{-Mon|Prs-Cte} <br>
+    Volt_12V_1{-Mon|Prs-Cte} <br>
+    S_Fan_1{-Mon|Prs-Cte} <br>
+    S_Fan_2{-Mon|Prs-Cte} <br>
+    S_Fan_3{-Mon|Prs-Cte} <br>
+    S_Fan_4{-Mon|Prs-Cte} <br>
+    S_Fan_5{-Mon|Prs-Cte} <br>
+    S_Fan_6{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>CU_Common</summary>
+    AutoAdjust-Cte <br>
+    FanLevel-SP <br>
+    FanLevel-RB <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    MState-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+    Pwr <br>
+<br>
+</details>
+
+# AMC
+## `$(AREA):$(DIS)-AMC_{1,3..12}:`
+<details>
+<summary>sensors</summary>
+    {Volt|Curr}_FMC{1|2}_12V{-Mon|Prs-Cte} <br>
+    {Volt|Curr}_FMC{1|2}_VADJ{-Mon|Prs-Cte} <br>
+    {Volt|Curr}_FMC{1|2}_3V3{-Mon|Prs-Cte} <br>
+    Temp_FPGA{-Mon|Prs-Cte} <br>
+    Temp_UC{-Mon|Prs-Cte} <br>
+    Temp_ClkSwitch{-Mon|Prs-Cte} <br>
+    Temp_DCDC{-Mon|Prs-Cte} <br>
+    Temp_RAM{-Mon|Prs-Cte} <br>
```

Comment:
We can use the "_" only when we have a misunderstanding case, for example for numbers, FMC1_12V... if it is not the case, we can avoid using it. Anyway, for this case, we can also use 
`FMC{1|2}{Volt|Curr}12V{-Mon|Prs-Cte}`